### ISSUE TITLE
Terrain Option

### ISSUE THREAD

**Issue by NovaSDF** [saltysimulations/salty-747#444]
**Is your feature request related to a problem? If so, please describe.**
Not really a problem, more of a missing feature.

**Describe the feature you want implemented.**
The Terrain feature does not work when selected. It would be great to see this implemented in a future update. Very nerve wracking landing in an unfamiliar mountainous airport in dead of night with no radar to show surrounding terrain.

**References**
None

**Additional context**
None
Your Discord:

**Comment by imranalli87** [comment]
Yes please add the terrain feature. It works on the  default 747 but not on Salty. I thought it was a bug but it's just a missing feature?

**Comment by DesertRick13** [comment]
post from Giridhar [Z+5:30] — 05/18/2022
https://discordapp.com/channels/698720578055700650/698720667843166208/975074545318887424   . This is to re-enable the map/terrain feature. I found this in the support section for the time around May 18. It was made clear in that section there would be no support for this, so  it's "at your own risk." I have tried it and use it on a regular basis.
